blocks/bricks/__init__.py

```diff
@@ -658,10 +658,17 @@ def add_property(f):
         return add_property
 
     def __getattr__(self, attr):
+        # Prevents infinite recursion when no attribute "f" is available
+        # (happens during serialization under unknown circumstances)
+        try:
+            self_f = self.__getattribute__("f")
```

Comment:
Well, but sometimes I do need the `f` value. That's why I first check that this value is available, otherwise the call `self.f` leads to infinite recursion.